pkg/network/ebpf/c/protocols/tls/go-tls-conn.h

```diff
@@ -108,10 +120,9 @@ static __always_inline bool __tuple_via_tcp_conn(tls_conn_layout_t* cl, void* tc
         return false;
     }
 
-    output->metadata = CONN_TYPE_TCP;
-    if (family == AF_INET6) {
+    if (family == AF_INET6 && output->saddr_h != 0 && output->saddr_l != 0) {
```

Comment:
Can you add documentation explaining the reason for the addition of the case where saddrl/h is 0?

Reply:
changed the behavior to match `read_conn_tuple_partial`